src/appsignal/cli.py

```diff
@@ -135,3 +150,54 @@ def _write_file(self):
             )
 
             f.write(file_contents)
+
+
+class DemoCommand(AppsignalCLICommand):
+    def __init__(self, push_api_key=None, application=None):
+        self._push_api_key = push_api_key or os.environ.get("APPSIGNAL_PUSH_API_KEY")
+        self._name = application
+
+    def run(self) -> int:
+        print("Sending example data to AppSignal...")
+
+        if self._push_api_key is None:
+            self._input_push_api_key()
+
+        if self._name is None:
+            self._input_name()
+
+        print()
+
+        client = Client(
+            active=True,
+            name=self._name,
+            push_api_key=self._push_api_key,
+            log_level="trace",
+        )
+        print(f"Starting AppSignal client for {self._name}...")
+        client.start()
+
+        tracer = trace.get_tracer(__name__)
+
+        # Performance sample
+        with tracer.start_as_current_span("HTTP GET") as span:
+            span.set_attribute("http.method", "GET")
+            span.set_attribute(
+                "appsignal.request.parameters",
+                json.dumps({"GET": {"id": 1}, "POST": {}}),
+            )
+            span.set_attribute(
+                "otel.instrumentation_library.name",
+                "opentelemetry.instrumentation.wsgi",
+            )
+
+        # Error sample
+        with tracer.start_as_current_span("HTTP GET") as span:
+            span.set_attribute("http.method", "GET")
+            span.set_attribute(
+                "appsignal.request.parameters",
+                json.dumps({"GET": {"id": 1}, "POST": {}}),
+            )
+            span.record_exception(ValueError("Something went wrong"))
```

Comment:
If the exception is not raised, it will not have a traceback.

```suggestion
            try:
                raise ValueError("Something went wrong")
            except ValueError as e:
                span.record_exception(e)
```